runtime/manta/src/lib.rs

```diff
@@ -224,7 +246,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	/// Relay Chain `TransactionByteFee` / 10
```

Comment:
This comment is redundant, basically the same as the code.

Reply:
No.